benchmarks/benchmark.js

```diff
@@ -85,25 +85,12 @@ for (let setup of setups) {
         }
     });
 
-    if (setup.name === 'ls') {
-        // Test without the OT interface indirection
-        s.bench(`startSpan imp (${setup.name})`, (N, t) => {
-            let tracer = createNonReportingTracer();
-            let tracerImp = tracer.imp();
-            t.start();
-            for (let i = 0; i < N; i++) {
-                let spanImp = tracerImp.startSpan('test');
-                spanImp.finish();
-            }
-        });
-    }
-
     s.bench(`span with logs (${setup.name})`, (N, t) => {
         let tracer = setup.makeTracer();
         t.start();
         for (let i = 0; i < N; i++) {
             let span = tracer.startSpan('test');
-            span.logEvent('Hello world!');
+            span.log({ event : 'log', message : 'Hello world!' });
```

Comment:
IMO we can omit `event: 'log'` here and below... `event` was intended for things like zipkin `cr`/`cs`/etc annotations.